coordinator/src/db/channels.rs

```diff
@@ -144,6 +144,9 @@ impl From<ln_dlc_node::channel::ChannelState> for ChannelState {
             ln_dlc_node::channel::ChannelState::Announced => ChannelState::Announced,
             ln_dlc_node::channel::ChannelState::Pending => ChannelState::Pending,
             ln_dlc_node::channel::ChannelState::Open => ChannelState::Open,
+            // mapping `ChannelState::OpenUnpaid` to open as the coordinator references the payment
+            // through the payment hash. Hence no need to reflect that state twice.
+            ln_dlc_node::channel::ChannelState::OpenUnpaid => ChannelState::Open,
```

Comment:
But then I would need to make sure that the channel gets from `OpenUnpaid` to `Open`. I wanted to avoid that as it would be additional potential failure prune code, that has no real value does it?